aspnetcore/get-started.md

```diff
@@ -0,0 +1,262 @@
+
+## Prerequisites
+
+Obtain and install the latest .NET SDK at [Download .NET](https://dotnet.microsoft.com/download/dotnet).
+
+## Create a Blazor Web App
+
+Open a command shell to a suitable location for the sample app and use the following command to create a Blazor Web App. The `-o|--output` option creates a folder for the project and names the project `BlazorSample`:
+
+```dotnetcli
+dotnet new blazor -o BlazorSample
+```
+
+## Run the app
+
+Change the directory to the `BlazorSample` folder with the following command:
+
+```dotnetcli
+cd BlazorSample
+```
+
+The `dotnet watch` command runs the app and opens your default browser to the app's landing page:
+
+```dotnetcli
+dotnet watch
+```
+
+:::image source="get-started/static/blazor-web-app-running.png" alt-text="Blazor Web App running in Microsoft Edge with the homepage rendered in the UI.":::
+
+Using the app's sidebar navigation, visit the Counter page, where you can select the **:::no-loc text="Click me":::** button to increment the counter.
+
+:::image source="get-started/static/blazor-web-app-counter-page-incremented-to-one.png" alt-text="Counter page rendered after the 'Click me' button is selected once, showing the counter incremented to a value of one.":::
+
+## Change the app
+
+Leave the browser open with the Counter page loaded. By using the `dotnet watch` command to run the app, you can make changes to the app's markup and code without having to rebuild the app to reflect the changes in the browser.
+
+The `Counter` Razor component that renders the Counter web page is located at `Components/Pages/Counter.razor` in the project.
+
+Open the `Counter.razor` file in a text editor and note a few interesting lines that render content and make the component's counter feature work.
+
+`Components/Pages/Counter.razor`:
+
+```razor
+@page "/counter"
+
+<PageTitle>Counter</PageTitle>
+
+<h1>Counter</h1>
+
+<p role="status">Current count: @currentCount</p>
+
+<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+
+@code {
+    private int currentCount = 0;
+
+    private void IncrementCount()
+    {
+        currentCount++;
+    }
+}
+```
+
+The file starts with a line that indicates the component's relative path (`/counter`):
+
+```razor
+@page "/counter"
+```
+
+The title of the page is set by `<PageTitle>` tags:
+
+```razor
+<PageTitle>Counter</PageTitle>
+```
+
+An H1 heading is displayed to the user:
+
+```razor
+<h1>Counter</h1>
+```
+
+A paragraph element (`<p>`) displays the current count, which is stored in a variable named `currentCount`:
+
+```razor
+<p role="status">Current count: @currentCount</p>
+```
+
+A button (`<button>`) allows the user to increment the counter, which occurs when a button click executes a C# method named `IncrementCount`:
+
+```razor
+<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+```
+
+C# code is present in the `@code` block:
+
+```razor
+@code {
+    ...
+}
+```
+
+Within the `@code` block:
```

Comment:
The "code block" without code is confusing, as is the apparent differentiation between "in" and "within"